doc/ref_model/chapters/chapter09.md

```diff
@@ -243,3 +255,138 @@ auto.cicd.014 | The CD pipeline must be able to automatically promote software f
+
+Common supporting system (OSS – Operation Support System, BSS – Business Support System) requirements are redefined, driven by introduction of new technologies in computing infrastructure and modern data centres with abstraction of resources – known as virtualization and cloud computing. This brings many advantages – such as easy scaling, error recovery, reaching a high level of operational autonomy etc., but also many new challenges in the Telecom network management space. Those novel challenges are mostly directed towards the dynamical nature of the system, orientation towards microservices instead of a silo approach, and huge amounts of data which has to be processed in order to understand the internal status of the system. Hence the need of improved ways to monitor systems - observability.
+
+
+<a name="9.6.1"></a>
+### 9.6.1. Why Observability
+
+Knowing the status of all services and functions at all levels in a cloud based service offering is essential to act fast, ideally pro-actively before users notice and, most importantly, before they call the help desk. 
+
+Common approach to understanding aforementioned telecom network status in conventional non-cloud environments is referred to as monitoring. Usually it would include metric information related toresources, such as CPU, memory, HDD, Network I/O, but also business related technical indicators (KPIs) such as number of active users, number of registrations, etc. This monitoring data isrepresented as a time series, retrieved in regular intervals, usually with granulation of 5 to 30 minutes.In addition to it, async messages such as alarms and notifications were exposed by the monitored systems in order to provide information about foreseen situations. It is worth noting that metric data provides approximation of the health of the system, while the alarms and notifications try to bring more information about the problem. In general, they provide information about known unknowns - anticipated situations occurring at random time. However, this would very rarely be sufficient information for understanding the problem (RCA - root cause analysis), therefore it is necessary to retrieve more data related to the problem - logs and network signalization. Logs are application output information created by developers to get more granular information about the code execution. Network packet captures/traces are useful since telecommunication networks are distributed systems where components communicate utilizing various protocols, and the communication can be examined to get details of the problem. The latter two sources of information need manual human analysis in order to extract useful information, and as such are usually not essential parts of the regular monitoring systems.
```

Comment:
The last sentence, "The latter two sources of information need manual human analysis in order to extract useful information, and as such are usually not essential parts of the regular monitoring systems." is not essentially correct -- there are multiple efforts to create a tracing system, including "[OpenTrace](https://opentracing.io/)".

```suggestion
Common approach to understanding aforementioned telecom network status in conventional non-cloud environments is referred to as monitoring. Usually it would include metric information related to resources, such as CPU, memory, HDD, Network I/O, but also business related technical indicators (KPIs) such as number of active users, number of registrations, etc. This monitoring data is represented as a time series, retrieved in regular intervals, usually with granulation of 5 to 30 minutes. In addition, async messages, such as alarms and notifications, are exposed by the monitored systems in order to provide information about foreseen situations. It is worth noting that metric data provides approximation of the health of the system, while the alarms and notifications try to bring more information about the problem. In general, they provide information about known unknowns - anticipated situations occurring at some random time. However, this would very rarely be sufficient information, such as the context, for understanding the problem (RCA - root cause analysis). Therefore it is necessary to retrieve more data related to the problem - logs and network signalization. Logs are application output information created by developers to trace the code execution flow. Network packet captures/traces are useful since telecommunication networks are distributed systems where components communicate utilizing various protocols, and the communication can be examined to get details of the problem. The latter two sources of information need manual human analysis in order to extract useful information, and as such are usually not essential parts of the regular monitoring systems.
```

Reply:
I was expecting to be challenged on that sentence, but because of something else. :)
So, I was referring to network packet capturing and logs. They are extensively used, but not for "normal" monitoring, rather for deeper investigations in case of problems. OpenTrace is talking about tracing in the sense of tracking distributed requests and responds of the services. But, I would agree that calls can be thought of as set of distributed requests, therefore it could be understood as a "tracing" in the sense provided by OpenTracing project. Anyway, I believe that the statements still holds: such tracing as proposed by OpenTracing is not part of current monitoring solutions (to the best of my knowledge).
I was expecting to be challenged exactly because I am saying that network capturing is not essential part of monitoring, but maybe some of the Telcos do have them deeply integrated.
In order to avoid possible different interpratations, I would change the sentence.

Reply:
As I wanted to propose another sentence, it occured to me that this (last) sentence could be completely removed. I am not sure if would be allowed to make an edit, in the worst case @sukhdevkapur would need to do so.

Reply:
Removed the last sentence